submissions/vovan-zt/Frogger-Game/app.js

```diff
@@ -0,0 +1,108 @@
+const gridSize = {
+    width: 101,
+    height: 81,
+};
+
+const fieldSize = {
+width: gridSize.width*5,
+height: gridSize.height*6,
+top:0,
+left:0,
+};
```

Comment:
Please fix formatting across the entire code base. Tune up your code editor do this for you. Use prettier.io plugin if code editor doesn't have built-in auto-formatting features or prettier plugin pre-installed.

Reply:
I agree, you need to install something to check spelling, I constantly make such mistakes